module-services/service-desktop/endpoints/include/endpoints/EndpointType.hpp

```diff
@@ -21,9 +21,10 @@ namespace sdesktop::endpoints
         calendarEventsPlaceholder,
         developerMode,
         bluetooth,
-        usbSecurity
+        usbSecurity,
+        outbox
     };
 
-    inline constexpr auto lastEndpoint = static_cast<int>(EndpointType::usbSecurity);
+    inline constexpr auto lastEndpoint = static_cast<int>(EndpointType::outbox);
```

Comment:
just saying - magic_enum in the repo has the capability to get enum length. I do not know if I would advise using it in the header (as it's heavy on compilation) but it's nice to know :)